api/dsap/relationships.js

```diff
@@ -1,22 +1,67 @@
 'use strict';
 module.exports = function (context) {
     var relationships = context.models.relationships;
+    var models = context.models;
 
-    return {
+    const functions = {
+        mapArtistToSubject: function (aritistId) {
+            // TODO Map artist to subject.
+            return 'A0EEBC99-9C0B-4EF8-BB6D-6BB9BD380A11';
+        },
+        mapWorkToSubject: function (workId) {
+            // TODO Map work to subject.
```

Comment:
Something like this should work:
```javascript
models.artists.findById(artistId)
  .then(artist => {
    return artist.getEntity();
  })
  .then(entity => {
    return entity.getRelEntity();
  });
```
This is pretty ugly as it creates three queries.

There should also be another way, like this:
```javascript
relEntities.findOne({
                attributes: ['id'],
                include: [{
                    model: entities,
                    include: [{
                        model: models.artists
                    }]
                }],
                where: {
                    '$artists.id$': artistId
                }
            })
```

This already includes some sequelize magic with the `$...$` which allows one to build a where clause on an included table, which is documented nowhere except for some Github issue. But it doesn't work and the query grows pretty large.

Maybe some Sequelize magician can jump in and fix this abomination of query...